build/int.cloudbuild.yaml

```diff
@@ -94,6 +94,27 @@ steps:
     - verify safer-cluster-local
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'cft test run TestSaferCluster --stage destroy --verbose']
+- id: init autopilot-private-firewalls
+  waitFor:
+    - create-all
+    - destroy simple-autopilot-private-local
```

Comment:
might need to move these to after the actual `destroy simple-autopilot-private-local` setp

Reply:
`invalid build: invalid .steps field: build step https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/16 - "init autopilot-private-firewalls" depends on "destroy simple-autopilot-private-local", which has not been defined`

Reply:
oh it is somewhat sequential, I see.